src/core/errorHandler.js

```diff
@@ -38,7 +38,17 @@ export class ErrorHandler {
     return (
       <ul className="ErrorHandler-list">
         {this.capturedError.messages.map(
-          (msg) => <li className="ErrorHandler-list-item">{msg}</li>
+          (msg) => {
+            let msgString = msg;
+            if (typeof msgString === 'object') {
+              // This is an unlikely scenario where an API response
```

Comment:
Ah, okay, I wasn't sure if it was something that happened but was super-rare. Just being protective of the possible bad future = 👍